activesupport/test/core_ext/enumerable_test.rb

```diff
@@ -86,6 +130,40 @@ def test_array_sums
     payments = [ SummablePayment.new(5), SummablePayment.new(15) ]
     assert_equal SummablePayment.new(20), payments.sum
     assert_equal SummablePayment.new(20), payments.sum { |p| p }
+
+    sum = GenericEnumerable.new([3, 5.quo(1)]).sum
```

Comment:
Sorry, I need to stop using GenericEnumerable here.
I'm correcting it now.
Please wait a moment.